tests/formats/cif.cpp

```diff
@@ -0,0 +1,107 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
+#include <streambuf>
+#include <fstream>
+
+#include "catch.hpp"
+#include "helpers.hpp"
+#include "chemfiles.hpp"
+using namespace chemfiles;
+
+TEST_CASE("Read files in CIF format") {
+    SECTION("Read single step") {
+        // This is how I imagine most people will resolve the conflict between
+        // CIF files and mmCIF files.
+        auto file = Trajectory("data/cif/1544173.cif", 'r', "CIF");
```

Comment:
It looks like the CIF reader is able to read mmCIF files, but I guess it will ignore any kind of secondary structure information. We should send a warning in this case.